_posts/en/newsletters/2022-03-23-newsletter.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 'Bitcoin Optech Newsletter #192'
+permalink: /en/newsletters/2022/03/23/
+name: 2022-03-23-newsletter
+slug: 2022-03-23-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes discussion about the speedy trial soft
+fork activation mechanism and links to an update of an optimized LN
+pathfinding algorithm.  Also included are our regular sections with
+descriptions of recent changes to services and client software,
+announcements of new releases and release candidates, and summaries of
+notable changes to popular Bitcoin infrastructure software.
+
+## News
+
+- **Speedy trial discussion:** a mention of the speedy trial [soft fork
+  activation method][topic soft fork activation] from the summary of a
+  recent meeting about the proposed [OP_CHECKTEMPLATEVERIFY][topic
+  op_checktemplateverify] opcode was spun off into a separate thread
+  on the Bitcoin-Dev mailing list for additional discussion after Jorge
+  Timón [expressed][timon st] concern about the use of speedy trial for
+  a soft fork he thought was controversial.
+
+    Russell O'Connor [explained][oconnor st] how the concerns had been
+    previously addressed.  Anthony Towns further [described][towns st]
+    how an unwanted soft fork activation using speedy trial could be
+    resisted by objecting users.
+
+- **Pathfinding algorithm update:** René Pickhardt [posted][pickhardt
```

Comment:
While I understand that many people historically call this path finding (very unfortunately we even do this in mastering the lightning network) or routing algorithms I think as a community we would do ourselves a favor if start using more accurate terms. 

Payment delivery on the lightning network is not a classical path finding problem but rather a transportation problem or (min cost) flow problem. We want to deliver or transport sats via a network of channels from a source to a sink. Of course flows may be dissected into paths or may be built up from several  paths or consist only of single path (mostly if the payment amount is small) but the notion of path finding seems more and more confusing.

Here are some suggestions
* Update on Payment delivery 
* Update on Payment flows
* Update on the runtime of optimally reliable payment flows
* Update on Multi Part Payments
* Update on efficiently computing the split for multi part payments.
* Update on splitting payments

My comment sound a bit nit-picky but I think conflicting this with path finding makes us and new people in the space a bit operationally blind. From the feedback I get I have the impression that even some experts struggle to understand that this is more about computing a good mpp split than actual path finding (though finding the paths is certainly part of the solution). In that sense I think even "Update on Pickhardt Payments" (how my co-author and many others have called our results) would be more precise than update on path finding.